@commitlint/load/src/utils/load-config.ts

```diff
@@ -11,7 +12,23 @@ export async function loadConfig(
 	cwd: string,
 	configPath?: string
 ): Promise<LoadConfigResult | null> {
-	const explorer = cosmiconfig('commitlint');
+	const moduleName = 'commitlint';
+	const explorer = cosmiconfig('commitlint', {
```

Comment:
This here can be the variable `moduleName`

```suggestion
	const explorer = cosmiconfig(moduleName, {
```